Source/Scene/Scene.js

```diff
@@ -2562,6 +2562,9 @@ define([
 
         us.update(frameState);
 
+        var backgroundColor = defaultValue(scene.backgroundColor, Color.BLACK);
```

Comment:
Ah true, someone could easily set it to undefined. It probably won't happen but no harm in keeping the old behavior.